lib/lambda_ethereum_consensus/state_transition/misc.ex

```diff
@@ -130,4 +133,130 @@ defmodule LambdaEthereumConsensus.StateTransition.Misc do
     # Converts an unsigned integer value to a bytes 4 value
     <<value::unsigned-integer-little-size(32)>>
   end
+
+  @spec uint64_to_bytes(SszTypes.uint64()) :: <<_::64>>
+  def uint64_to_bytes(value) when is_integer(value) and value >= 0 do
+    <<value::unsigned-integer-little-size(64)>>
+  end
+
+  @spec compute_committee(
+          list(SszTypes.validator_index()),
+          SszTypes.bytes32(),
+          SszTypes.uint64(),
+          SszTypes.uint64()
+        ) ::
+          {:ok, list(SszTypes.validator_index())} | {:error, binary()}
+  def compute_committee(indices, seed, index, count) do
+    start_ = div(length(indices) * index, count)
```

Comment:
why `start_` and not `start`?

Reply:
That was for naming convention, since if I use `end` instead of `end_` at the next line it generates error.